test/OpBitcast_ptr_vector.spvasm

```diff
@@ -0,0 +1,34 @@
+; Check support of OpBitcast with pointer operands
+; Converts to vectors of integers, which is supported by SPIR-V 1.5
+
+; REQUIRES: spirv-as
+; RUN: spirv-as --target-env spv1.5 -o %t.spv %s
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis %t.rev.bc
+; RUN: FileCheck < %t.rev.ll %s --check-prefix=CHECK-LLVM
+               OpCapability Addresses
+               OpCapability Kernel
+               OpCapability Int64
+               OpMemoryModel Physical64 OpenCL
+               OpEntryPoint Kernel %kernel "test"
+       %uint = OpTypeInt 32 0
+      %ulong = OpTypeInt 64 0
```

Comment:
Seems unused.

Reply:
Thanks!  I adapted this test from the CTS tests I've been working on and this one slipped through.

Out of curiosity, are you using a tool to find these?

Reply:
> Out of curiosity, are you using a tool to find these?

Nope, just manual inspection. :-)